assets/scss/modules/controls/_condition-navigation.scss

```diff
@@ -0,0 +1,17 @@
+nav.condition-navigation {
```

Comment:
Do you need the element selector at the start? Can't it just be `.condition-navigation` ?